src/Future/functions.php

```diff
@@ -29,39 +29,82 @@ function race(iterable $futures, ?Cancellation $cancellation = null): mixed
     throw new \Error('No future provided');
 }
 
+/**
+ * Unwraps the first completed future.
+ *
+ * If you want the first future completed without an error, use {@see any()} instead.
+ *
+ * @template T
+ *
+ * @param iterable<Future<T>> $futures
+ * @param Cancellation|null $cancellation Optional cancellation.
+ *
+ * @return T
+ *
+ * @throws \Error If $futures is empty.
+ *
+ * @deprecated Use {@see awaitFirst()} instead.
+ */
+function race(iterable $futures, ?Cancellation $cancellation = null): mixed
+{
+    return awaitFirst($futures, $cancellation);
+}
+
 /**
  * Unwraps the first successfully completed future.
  *
- * If you want the first future completed, successful or not, use {@see race()} instead.
+ * If you want the first future completed, successful or not, use {@see awaitFirst()} instead.
  *
  * @template Tk of array-key
  * @template Tv
  *
  * @param iterable<Tk, Future<Tv>> $futures
- * @param Cancellation|null        $cancellation Optional cancellation.
+ * @param Cancellation|null $cancellation Optional cancellation.
  *
  * @return Tv
  *
  * @throws CompositeException If all futures errored.
  */
-function any(iterable $futures, ?Cancellation $cancellation = null): mixed
+function awaitAny(iterable $futures, ?Cancellation $cancellation = null): mixed
 {
-    $result = some($futures, 1, $cancellation);
+    $result = awaitN($futures, 1, $cancellation);
     return $result[\array_key_first($result)];
 }
 
+/**
+ * Unwraps the first successfully completed future.
+ *
+ * If you want the first future completed, successful or not, use {@see awaitFirst()} instead.
+ *
+ * @template Tk of array-key
+ * @template Tv
+ *
+ * @param iterable<Tk, Future<Tv>> $futures
+ * @param Cancellation|null $cancellation Optional cancellation.
+ *
+ * @return Tv
+ *
+ * @throws CompositeException If all futures errored.
+ *
+ * @deprecated Use {@see awaitFirst()} instead.
+ */
+function any(iterable $futures, ?Cancellation $cancellation = null): mixed
+{
+    return awaitAny($futures, $cancellation);
+}
+
 /**
  * @template Tk of array-key
  * @template Tv
  *
  * @param iterable<Tk, Future<Tv>> $futures
- * @param Cancellation|null        $cancellation Optional cancellation.
+ * @param Cancellation|null $cancellation Optional cancellation.
  *
  * @return non-empty-array<Tk, Tv>
  *
  * @throws CompositeException If all futures errored.
  */
-function some(iterable $futures, int $count, ?Cancellation $cancellation = null): array
+function awaitN(iterable $futures, int $count, ?Cancellation $cancellation = null): array
```

Comment:
Should we switch the parameter order here, so `$count` will be the first param?

Reply:
Agreed.

Reply:
Should this be named `awaitAnyN`?

Reply:
possibly there should be awaitN (= after N futures completed, we're done) and awaitAnyN (= after any N futures succeeded, we're done)?

Reply:
Yeah, discussed this with @trowski privately as well, we could introduce `awaitFirstN` later if needed.